wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/WorkWithMaps.wiki

```diff
@@ -0,0 +1,49 @@
+---
+Test
+---
+!1 Iteraction with maps using !-BrowserTest-! with Storyboard
+Sample test on [[Google Maps][https://www.google.com/maps]].
+
+!note '''Note:''' More about getting offsets in canvas maps: https://developers.google.com/maps/documentation/javascript/reference/overlay-view#MapCanvasProjection
+
+!*> definitions
+!define mapUrl {!-https://www.google.com/maps/@51.7147561,11.342505,5.16z-!}
+!define minimap {!-css=.widget-minimap-shim.widget-minimap-shim-button-!}
+!define map {css=[aria-label="Map"]}
+!define locationBottomPopup {css=div.widget-reveal-card-container}
+*!
+!*> libraries
+|library     |
+|browser test|
+*!
+!*> scenarios
+|scenario           |lets open canvas map|
+|set browser size to|1024   |by   |800   |
+|open               |${mapUrl}           |
+|wait for visible   |${minimap}          |
+|click              |Remind me later     |
+
+|scenario    |lets test zooming by double clicking with offset|
+|double click|${map}    |xoffset    |294   |yoffset   |-200   |
```

Comment:
Thanks for insights,

Will try my best to make it more readable and with assertions for example test ;)

Either way i decided to stay on offset parameters instead of latitude, longitude, even though those are a bit more tricky to get (requires different tools/plugins on different cases). There are several reasons:

- Some html elements are clickable only on certain pixels:
_I met html elements, which would only listen to event clicks at certain points of element itself. As an example, imagine png type image which would only listen to clicks on areas, that contain data.. ;)_ 
- To be compliant with most users use cases.
_It is possible to make click by longitude, latitude method, but it would only work for specific cases (owning, running and having adjusted google maps layers). But I assume, that wouldn't work for HERE, openstreet maps. P.s. i made it but it requires additional maintainance according chrome + google maps releases ;)_

- To be consistent, and not overload with methods. 
_I'm planning to add additional drag and drop method for dragging elements for offset for upcoming commit (imagine adjusting sound volume or intractable graphs automation) ;)_

Reply:
hanges done in https://github.com/fhoeben/hsac-fitnesse-fixtures/pull/286/commits/51284650074d64134012fbc2a91c496ae8723471